docs/using/security-practices.md

```diff
@@ -308,6 +295,6 @@ off, which is still a big win.
 Sandstorm will soon employ the `Content-Security-Policy` header to
 prevent an app from communicating with other origins without
 permission, in order to implement full confinement. As of this writing
-(May 2015), this has not yet been put in place, mostly because
-server-side confinement is not complete (as described earlier) which
-makes client-side confinement largely moot for the moment.
+(April 2021), this has been implemented, and can be enabled with a
+configuration option, but is pending wider rollout on ensuring existing
+apps are updated to be compatible with this change.
```

Comment:
Note that the version that's implemented is still not quite full confinement. In particular:

- Images & media are still allowed through; I have some WIP that rigs up csp reporting to trigger permission prompt rather than just letting it through.
- It is still possible to use webrtc, and as of right now there's no way to tell the browser to disallow this -- I need to get back to bugging w3c folks about this.

Reply:
Newest commit should address this.